cff/README.md

```diff
@@ -0,0 +1,14 @@
+# /cff/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for Citation File Format (CFF) resources.
+
+## Uses
+Currently, the namespace generally redirects to the website, while `application/schema+json` and URL patterns `^schema([^/]+/?)$` are forwarded to the latest and related JSON schema version respectively.
+
+## Contact
+This space is administered by:
+
+*Stephan Druskat*
+Berlin, Germany
+<stephan.druskat@dlr.de>
+GitHub: [sdruskat](https://github.com/sdruskat)
+ORCID: [0000-0003-4925-7248](https://orcid.org/0000-0003-4925-7248)
```

Comment:
Line breaks are clearly desired, but not preserved by current markdown. Therefore —
```suggestion
*Stephan Druskat* <br>
Berlin, Germany <br>
<stephan.druskat@dlr.de> <br>
GitHub: [sdruskat](https://github.com/sdruskat) <br>
ORCID: [0000-0003-4925-7248](https://orcid.org/0000-0003-4925-7248)
```

Reply:
Thanks @TallTed, good point, I forgot to add the linebreaks (I prefer doing them with double spaces rather than HTML tags). Now added in  #4410.

Reply:
Sure. I prefer the `<br>` because they are visible, but double-spaces have the same result.